### PR TITLE
fix(teams_playbook): replace PATCH -> POST

### PR DESCRIPTION
**Pull request by jasonpagetas**
POST is required to overwrite

- relate: https://github.com/nofusscomputing/ansible_collection_centurion/issues/29

### CODE REVIEW

**Comment by jasonpagetas** [comment]
@jon-nfc This is good to go for a code review and merging. Everything works as intended now

File: docs/projects/ansible/collection/centurion/playbooks/teams.md [hunk 1]
```diff
@@ -21,14 +21,14 @@ The following job template will be created:
...
+The teams playbook gathers information regarding centurion organisations from the ansible inventory. Using this information the play is designed to create new teams with permissions and notes. The workflow for the playbook is as follows
 
 - Fetch all organisations from Centurion ERP
 - Fetch all existing teams within each organisation from Centurion ERP
 - Fetch any teams to be created from inventory
 - Create new teams
-- Patch all teams with required permissions
-- Patch all teams with required notes
+- POST all teams with required permissions
+- POST all teams with required notes
```
Comment (jon-nfc) [inline docs/projects/ansible/collection/centurion/playbooks/teams.md]:
```suggestion
- for each team

    - Replace the team permissions

    - Replace the contents of the notes field
```

File: docs/projects/ansible/collection/centurion/playbooks/teams.md [hunk 2]
```diff
@@ -21,14 +21,14 @@ The following job template will be created:
 
 ## Play workflow
 
-The teams playbook gathers information regarding centurion organisations from the ansible inventory. Using this information the play is designed to create new teams, patch permissions and patch notes. The workflow for the playbook is as follows
+The teams playbook gathers information regarding centurion organisations from the ansible inventory. Using this information the play is designed to create new teams with permissions and notes. The workflow for the playbook is as follows
 
 - Fetch all organisations from Centurion ERP
 - Fetch all existing teams within each organisation from Centurion ERP
 - Fetch any teams to be created from inventory
 - Create new teams
```
Comment (jon-nfc) [inline docs/projects/ansible/collection/centurion/playbooks/teams.md]:
```suggestion
- Fetch all organisations from Centurion ERP

- Fetch all existing teams within each organisation from Centurion ERP

- Fetch any teams to be created from inventory

- Create new teams

```

spacing..

File: playbooks/teams.yaml [hunk 3]
```diff
@@ -219,7 +219,7 @@
           {{ nfc_pb_disable_log | default(true) }}
 
 
-    - name: Patch team notes
+    - name: replace team notes
```
Comment (jon-nfc) [inline playbooks/teams.yaml]:
```suggestion
    - name: Replace team notes
```

**Review by jon-nfc — CHANGES_REQUESTED** [review]
ci jobs now run however the report step failed. to view the fails for liniting, download the mkdocs-lint artifact and open with a text editor.

**Review by jon-nfc — CHANGES_REQUESTED** [review]
.

**Review by jon-nfc — APPROVED** [review]
GTG

**Pull request merged**